install/generator/09-syndesis-public-oauth-proxy.yml.mustache

```diff
@@ -0,0 +1,184 @@
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: syndesis-public-oauthproxy
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-public-oauthproxy
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
```

Comment:
@rhuss @heiko-braun this brings up the issue that this proxy and the public API should really only be installed by someone who has cluster level privileges. Looking at it another way, since the API exposes a special endpoint that allows long lived service account token access, it should be something that a cluster admin is aware of. 
Any ideas on how we can make this an optional thing in syndesis install, so only a user with the right privileges would install this?

Reply:
I agree with the concerns raised about this. I'm refactoring the public proxy out of the installer into its own template to be run separately to create a public proxy with required permissions.

Reply:
Moved public proxy out into it's own template `syndesis-public-oauthproxy`, also added instructions in README.md on how to install using the template.